addons/languages/rusyn/apk/build.gradle

```diff
@@ -0,0 +1,8 @@
+//owner unknown (may have not been published yet)
+ext.shouldBePublished = false
+
+apply from: "$rootDir/gradle/pack_apk.gradle"
```

Comment:
this should be: `apply from: "$rootDir/addons/gradle/pack_apk.gradle"`

Reply:
Fixed in commit bf9abef Resolved relative paths and unique id